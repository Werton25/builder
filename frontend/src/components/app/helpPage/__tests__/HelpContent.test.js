import React from "react";
import renderer from "react-test-renderer";
import HelpContent from "../HelpContent";

describe('HelpContent tests', () => {
  it('HelpContent renders correctly', () => {
    const tree = renderer
      .create(<HelpContent/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});