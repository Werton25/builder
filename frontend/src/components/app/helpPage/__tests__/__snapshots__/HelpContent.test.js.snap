// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpContent tests HelpContent renders correctly 1`] = `
<table
  className="table"
>
  <thead>
    <tr>
      <th>
        Сочетание клавиш
      </th>
      <th>
        Описание
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        Ctrl + Enter
      </td>
      <td>
        Включить полноэкранный режим
      </td>
    </tr>
    <tr>
      <td>
        Ctrl + F
      </td>
      <td>
        Поиск по тексту
      </td>
    </tr>
    <tr>
      <td>
        Ctrl + Alt + L
      </td>
      <td>
        Отформатировать текст
      </td>
    </tr>
  </tbody>
</table>
`;
